packages/tooltip/src/context.ts

```diff
@@ -0,0 +1,45 @@
+import { createContext, MouseEvent } from 'react'
+import { TooltipAnchor } from './types'
+
+export interface TooltipActionsContextData {
+    showTooltipAt: (
+        content: JSX.Element,
+        position: [number, number],
+        anchor?: TooltipAnchor
+    ) => void
+    showTooltipFromEvent: (content: JSX.Element, event: MouseEvent) => void
+    hideTooltip: () => void
+}
+
+const defaultActions: TooltipActionsContextData = {
+    showTooltipAt: () => {},
+    showTooltipFromEvent: () => {},
+    hideTooltip: () => {},
+}
+
+export const TooltipActionsContext = createContext<TooltipActionsContextData>(defaultActions)
```

Comment:
You could have not provided a default value for the contexts, since in the hooks you are checking for undefined values before returning the context value.

Reply:
Right.